site/layouts/_default/docs.html

```diff
@@ -16,9 +16,12 @@ <h5 class="offcanvas-title" id="bdSidebarOffcanvasLabel">Browse docs</h5>
     <main class="bd-main order-1">
       <div class="bd-intro pt-2 ps-lg-2">
         <div class="d-md-flex flex-md-row-reverse align-items-center justify-content-between">
-          <a class="btn btn-sm btn-bd-light mb-3 mb-md-0 rounded-2" href="{{ .Site.Params.repo }}/blob/v{{ .Site.Params.current_version }}/site/content/{{ .Page.File.Path | replaceRE `\\` "/" }}" title="View and edit this file on GitHub" target="_blank" rel="noopener">
-            View on GitHub
-          </a>
+          <div class="mb-3 mb-md-0 d-flex">
+            <small class="d-inline-flex px-2 py-1 fw-semibold text-success bg-success bg-opacity-10 border border-success border-opacity-10 rounded-2 me-2">Added in v{{ .Page.Params.added }}</small>
```

Comment:
I'd prefer to use the "added-in" shortcode but I haven't found a way to reuse it here. I'm not a Hugo grandmaster so if someone know how I can do that, feel free to add a comment around here.

Reply:
@XhmikosR for ideas :)

Reply:
Hmm, I don't think it's possible here because it's a layout file. I'd go with the current solution :)